p2p/netaddress_test.go

```diff
@@ -116,7 +116,7 @@ func TestNetAddressProperties(t *testing.T) {
 		routable bool
 	}{
 		{"127.0.0.1:8080", true, true, false},
-		{"ya.ru:80", true, false, true},
+		{"123fdc2a9bce5f2ef01e907a1d6a16548ee5a245@ya.ru:80", true, false, true},
```

Comment:
Can you add these as new vectors, instead of removing the old vector? (Unless the old test didn't make sense for some reason)